wp/lib/bap_wp/src/constraint.ml

```diff
@@ -208,35 +208,35 @@ let rec del_empty_constr_hyps (const : t) : t =
     let fold_list c_list =
       List.fold_right c_list ~init:[]
         ~f:(fun x y ->
-          match x with
-          | Clause([],xs) ->
-             List.append (List.map xs ~f:del_empty_constr_hyps) y
-          | _ -> (del_empty_constr_hyps x) :: y) in
+            match x with
```

Comment:
Are we ocp-indent compliant?

Reply:
I have a git hook that runs ocp-indent on all the files I edit. That's the source of all these whitespace changes.

Reply:
Great, I'll just ignore whitespace from now on in the commits.